tabcmd/commands/datasources_and_workbooks/datasources_and_workbooks_command.py

```diff
@@ -60,39 +62,46 @@ def get_ds_by_content_url(logger, server, datasource_content_url) -> TSC.Datasou
         return matching_datasources[0]
 
     @staticmethod
-    def apply_values_from_url_params(request_options: TSC.PDFRequestOptions, url, logger) -> None:
+    def apply_values_from_url_params(logger, request_options: TSC.PDFRequestOptions, url) -> None:
         # should be able to replace this with request_options._append_view_filters(params)
         logger.debug(url)
         try:
             if "?" in url:
                 query = url.split("?")[1]
-                logger.trace("Query parameters: {}".format(query))
+                logger.debug("Query parameters: {}".format(query))
             else:
                 logger.debug("No query parameters present in url")
                 return
 
             params = query.split("&")
-            logger.trace(params)
+            logger.debug(params)
             for value in params:
                 if value.startswith(":"):
-                    DatasourcesAndWorkbooks.apply_option_value(request_options, value, logger)
+                    DatasourcesAndWorkbooks.apply_option_value(logger, request_options, value)
                 else:  # it must be a filter
-                    DatasourcesAndWorkbooks.apply_filter_value(request_options, value, logger)
+                    # the REST API doesn't appear to have the option to disambiguate with "Parameter.<fieldname>"
```

Comment:
Just double checking. The comment says `Parameter.` but the code uses the plural `Parameters.`